### PR TITLE
Bugfix/regression

### PR DESCRIPTION
**Pull request by priyavadan**
Fixes Issue #52 and #53

### CODE REVIEW

File: bin/origen [hunk 1]
```diff
@@ -134,16 +136,37 @@ if origen_root && File.exist?(ENV['BUNDLE_GEMFILE']) && Origen.site_config.gem_m
                   (spec.executables || []).each do |bin|
                     FileUtils.cp("#{spec.bin_dir}/#{bin}", "#{local_gem_dir}/bin")
                   end
-                  FileUtils.cp(spec.cache_file, "#{local_gem_dir}/cache")
+                  if File.exist(spec.cache_file)
```
Comment (ginty) [inline bin/origen]: Don't think it needs to be so noisy when things are going OK, I would remove all the puts statements

Comment (priyavadan) [inline bin/origen]: updated

**Comment by ginty** [comment]
LGTM, thanks!

**Comment by priyavadan** [comment]
Thanks! origen_core 0.7.13 has been released.

**Pull request merged**